tests/testsrc/test262.properties

```diff
@@ -2312,25 +2312,15 @@ built-ins/Symbol 36/94 (38.3%)
     asyncDispose/prop-desc.js
     asyncIterator/prop-desc.js
     dispose/prop-desc.js
-    for/cross-realm.js
```

Comment:
why does the overall count does not change? Have you regenerated the file or just removed the failing tests?